core/commands/partials/terraform_configuration_generator.rb

```diff
@@ -360,6 +376,7 @@ def security_group_resource(vpc)
           cidr_blocks = ["0.0.0.0/0"]
         }
       <% end %>
+      <%= tags_block %>
```

Comment:
It is good to have tags applied to all the things that connect this resource to the concrete configuration. But I do not see how these tags are used in the cleanup. Are they meant just for manual search in the AWS?